pkg/pluginmanager/manager.go

```diff
@@ -1659,3 +1659,119 @@ func getPluginDiscoveries() ([]configtypes.PluginDiscovery, error) {
 	discoverySources, _ := configlib.GetCLIDiscoverySources()
 	return append(discoverySources, testDiscoveries...), nil
 }
+
+// IsPluginsFromPluginGroupInstalled checks if all plugins from a specific group are installed and if a new version is available.
+// This function uses cache data to verify rather than fetching the inventory image
+func IsPluginsFromPluginGroupInstalled(name, version string) (bool, bool, error) {
+	// Disable logs.
+	log.QuietMode(true)
+	// Ensure that logs are re-enabled when we're done.
+	defer log.QuietMode(false)
```

Comment:
Same as above on providing a way to show the logs for debugging.

Reply:
added an env to control logs